proto/ignition/msgs/material.proto

```diff
@@ -128,4 +128,7 @@ message Material
   /// \brief If true, the mesh that this material is applied to will be
   /// rendered as double sided
   bool double_sided      = 12;
+
+  /// \brief Specular exponent
```

Comment:
The documentation is a bit confusing, because there's also a `specular` field above.